k8s/namespaces/ccd/ccd-api-gateway-web/demo.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ccd-api-gateway-web
+spec:
+  releaseName: ccd-api-gateway-web
+  values:
+    nodejs:
+      ingressClass: traefik-no-proxy
+      environment:
+        CORS_ORIGIN_WHITELIST: "*"
+        TIMING-ALLOW-ORIGIN: "https://www-ccd.demo.platform.hmcts.net"
```

Comment:
should should be set either in the base or in your helm chart and only overridden in prod

Reply:
@MSancaktutar i assume this should be in helm chart?

Reply:
And this in perf: 
```
CORS_ORIGIN_WHITELIST: "*"
TIMING-ALLOW-ORIGIN: "https://www-ccd.perftest.platform.hmcts.net"
UV_THREADPOOL_SIZE: 128
```

Reply:
We'll action clean up pf env vars from env-specific flux files under a separate ticket.